elements/howto-tabs/howto-tabs.js

```diff
@@ -284,24 +311,81 @@
       // changes its role to `tab`.
       this.setAttribute('role', 'tab');
       if (!this.id)
-        this.id = `howto-tabs-tab-generated-${dashTabCounter++}`;
+        this.id = `howto-tabs-tab-generated-${howtoTabCounter++}`;
+
+      // Set a well-defined initial state.
+      this.selected = false;
```

Comment:
I don't think you need to do this. It'll be `false` already because the getter is synced with the attribute. And if the user put `[selected]` on it, then it'll reflect that, which is the behavior we want I think.